frontend/packages/topology/src/__tests__/TopologySideBar.spec.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
 import { shallow } from 'enzyme';
 import { CloseButton } from '@console/internal/components/utils';
-import { TopologySideBar, TopologySideBarProps } from '../TopologySideBar';
+import TopologySideBar from '../components/side-bar/TopologySideBar';
 
 describe('TopologySideBar:', () => {
-  const props: TopologySideBarProps = {
```

Comment:
hmmm... this could be to our loss if the type changes shape.

Reply:
we'd get compile errors or test failures. A lot of tests don't use prop types.

Reply:
Ah okay, no worries.